diff --git a/scripts/build.js b/scripts/build.js
index e9e3ca0ab..4730d22be 100644
--- a/scripts/build.js
+++ b/scripts/build.js
@@ -237,6 +237,12 @@ export function configure({ arch, platform, out = join(ROOT, 'dist') }) {
         'package.json',
         'LICENSE*'
       ].join(',')}}`
+    },
+    download: {
+      checksums: {
+        "electron-v39.2.3-linux-arm64.zip": "c58c5904d6015cbbfa5f04fbda5c83b9a276a3565b5f3fa166795c789b055cdd",
+        "electron-v39.2.3-linux-x64.zip": "5eb51ebcb60487c4fc3a5b74ffb57a03eefd48def32200adf310ffaba4153d64"
+      }
     }
   }
 }
