diff --git a/scripts/build.js b/scripts/build.js
index e9e3ca0ab..e184c6036 100644
--- a/scripts/build.js
+++ b/scripts/build.js
@@ -237,6 +237,12 @@ export function configure({ arch, platform, out = join(ROOT, 'dist') }) {
         'package.json',
         'LICENSE*'
       ].join(',')}}`
+    },
+    download: {
+      checksums: {
+        "electron-v38.5.0-linux-arm64.zip": "d4939648feef4969d49eed845852a13a52ef0d0703bebcd20fade2a198a88080",
+        "electron-v38.5.0-linux-x64.zip": "b4c5b139a11816da7d703a6e2d03369ad1425e4fa5a8b6ed1d6cb42ce7775821"
+      }
     }
   }
 }
