diff --git a/src/main/args.js b/src/main/args.js
index 46ca87fed..57ef3b815 100644
--- a/src/main/args.js
+++ b/src/main/args.js
@@ -57,9 +57,8 @@ export function parse(argv = process.argv.slice(1)) {
     opts: program.opts(),
     args: program.args
       // TODO remove with allowUnknownOption!
-      .filter(arg => !arg.startsWith('-'))
-      .map(arg => argToURL(arg))
-
+      .filter(arg => !(arg.startsWith('-') || arg.trim().length === 0))
+      .map(arg => argToURL(arg.trim()))
   }
 }
 
